solutions/src/test/java/ru/sberbank/school/task02/FxClientControllerImplTest.java

```diff
@@ -0,0 +1,57 @@
+package ru.sberbank.school.task02;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.rules.ExpectedException;
+import ru.sberbank.school.task02.exception.ConverterConfigurationException;
+
+import static org.junit.Assert.*;
+
+public class FxClientControllerImplTest {
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Test
+    public void mainWithCorrectArgsDoesNotThrowsAnything() {
+        Assertions.assertDoesNotThrow(() ->
+            FxClientControllerImpl.main(new String[]{"-amount", "100", "-direction", "SELL", "-symbol", "USD/RUB"})
```

Comment:
&#39;FxClientControllerImpl&#39; has incorrect indentation level 12, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-455384056>*